### DISCUSSION TITLE
Combine library songs and liked songs

### DISCUSSION THREAD

**Discussion by z-huang** [z-huang/music#1387] [Ideas]
YouTube Music and Spotify now don't have library songs. There are only liked songs. What do you think? Would you like InnerTune to be in line with them, or stay as it is?

**Comment by Nolli20** [comment]
I have a question when I add my songs of playlist it add in inverse sense can you help me plz

**Reply by z-huang** [reply]
See #1225

**Comment by nvllz** [comment]
Yes, simplicity over complexity - I still don't know what the difference is between the two.

**Comment by siggi1984** [comment]
Personally, I think it's fine the way it is now. I would leave it.

**Comment by pdimu** [comment]
If it changes, all the library songs will need a conversion to liked songs

**Comment by teaegg0411** [comment]
Yes, I prefer combining library songs and liked songs together.

In Android Auto, it showes only library songs and albums. If you want to listen to an album, you have to add one of the songs to library. If you just like an album (which is more easy because there is a like button in album page) it won't show up in Android Auto.

**Comment by Montinevra** [comment]
I would change it so that library automatically includes all liked and downloaded songs. I like being able to add a song without have to like or download it, but I also want to be able to browse my entire collection at once.

**Comment by mattcarter11** [comment]
In my opinion I would combine it and simply keep the chips as filters that you can enable or disable. 

This could allow for cool thinks like filtering: liked and downloaded songs🤯
